File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/request/inventario-modificar-compra.request.ts
import { InventarioModificarCompraDetRequest } from "./inventario-modificar-compra-det.request";

export class InventarioModificarCompraRequest {
    idUsuarioMod: number;
    fecUsuarioMod: Date;
    listaDetalleInventario: InventarioModificarCompraDetRequest[];
}

File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/request/InventarioModificarCompraRequest.java
package com.besoft.siserp.lgtc.dto.request;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class InventarioModificarCompraRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idUsuarioMod;
	private Date fecUsuarioMod;
	private List<InventarioModificarCompraDetRequest> listaDetalleInventario;

}
